gordo_components/model/anomaly/diff.py

```diff
@@ -237,8 +278,8 @@ def anomaly(
         data["total-anomaly"] = np.linalg.norm(data["tag-anomaly"], axis=1)
 
         # If we have `thresholds_` values, then we can calculate anomaly confidence
-        if hasattr(self, "thresholds_"):
-            confidence = tag_anomaly.values / self.thresholds_.values
+        if hasattr(self, "feature_thresholds_"):
+            confidence = tag_anomaly.values / self.feature_thresholds_.values
```

Comment:
The `tag_anomaly` is this class defined as the absolute scaled residuals.

For each input value, we want the absolute _unscaled_ residuals to be considered.

Reply:
Right, this PR does not address that. We talked about how it use to be this way, (with unscaled) and then it was pushed that this should be _scaled_. We can flesh out what we're going to do there in another PR.